network.js

```diff
@@ -43,16 +43,15 @@ function setJavaProxyArgs(fields) {
     `-Dhttp.proxyHost=${fields.hostname}`,
     `-Dhttp.proxyPort=${fields.port}`,
     `-Dhttps.proxyHost=${fields.hostname}`,
-    `-Dhttps.proxyPort=${fields.port}`
+    `-Dhttps.proxyPort=${fields.port}`,
+    `-Dhttp.nonProxyHosts="localhost|127.0.0.1|0.0.0.0"`
```

Comment:
@tejohnso shouldn't we also exclude other local IP ranges like `10.*` as we do on the Electron side?